### PR TITLE
Prepare for throttling of replay.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

File: searchcore/src/vespa/searchcore/proton/common/replay_feedtoken_state.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+#pragma once
+
+#include "feedtoken.h"
+#include <vespa/vespalib/util/shared_operation_throttler.h>
+
+namespace proton::feedtoken {
+
+/*
+ * Feed token state used during replay.
```
Comment (geirst) [inline searchcore/src/vespa/searchcore/proton/common/replay_feedtoken_state.h:11]: Consider adding a comment regarding the wrapping of a throttler token, and what this implies.

**Review by geirst — APPROVED** [review]
👍

File: searchcore/src/vespa/searchcore/proton/common/replay_feedtoken_state.h (hunk 1 above)
Comment (toregge) [reply searchcore/src/vespa/searchcore/proton/common/replay_feedtoken_state.h]: Done

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**